src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts

```diff
@@ -1,29 +1,33 @@
-import type { MeshAttribute } from '@loaders.gl/schema';
+import type { MeshAttribute } from "@loaders.gl/schema";
 
 import { Vector3 } from "@math.gl/core";
 
 const scratchVector = new Vector3();
+const scratchVector2 = new Vector3();
 
 /**
  * Calculate triangle vertices of tile
  * @param attribute
  * @param offset
  */
-export const getTriangleVertices = (attribute: MeshAttribute, offset: number): Vector3[] => {
+export const getTriangleVertices = (
+  attribute: MeshAttribute,
+  offset: number
+): Vector3[] => {
   const geometryVertices: Vector3[] = [];
   for (let i = 0; i < 3; i++) {
     // @ts-expect-error - This expression is not constructable. Type 'Function' has no construct signatures.
     const typedArray = new attribute.value.constructor(3);
-    // console.log('typedArray----', typedArray);
     const subarray = attribute.value.subarray(
-      (offset + i) * attribute.size,
-      (offset + i) * attribute.size + attribute.size
+      offset + i * attribute.size,
```

Comment:
Formula is changed, is it correct?

Reply:
Yes. I've found a mistake when I fixed a test for this function.

Reply:
Yes, I've found a mistake when I fixed the retun type and the test